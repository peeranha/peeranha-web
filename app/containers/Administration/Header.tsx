import { useTranslation } from 'react-i18next';
import React from 'react';

import { MediumIconStyled } from 'components/Icon/MediumIcon';
import Wrapper from 'components/Header/Simple';
import Icon from 'components/Icon';
import H3 from 'components/H3';

import AddRoleForm from 'containers/Administration/AddRoleForm';
import { Moderator } from 'containers/Administration/types';

import AddModeratorButton from 'containers/Administration/AddModeratorButton';

import { singleCommunityColors } from 'utils/communityManagement';
import { BORDER_PRIMARY, ICON_TRASPARENT_BLUE } from 'style-constants';
import { Administration } from 'icons/index';
import { css } from '@emotion/react';

const colors = singleCommunityColors();

const customColor = colors.linkColor || BORDER_PRIMARY;

type HeaderProps = {
  locale: string;
  single?: number;
  moderators: Array<Moderator>;
  addRole: (userAddress: string, role: number, communityId: number) => void;
  addRoleLoading: boolean;
};

export const Header: React.FC<HeaderProps> = ({
  locale,
  single,
  moderators,
  addRole,
  addRoleLoading,
}): JSX.Element => {
  const { t } = useTranslation();

  return (
    <Wrapper className="mb-to-sm-0 mb-from-sm-3">
      <H3>
        <div
          css={css`
            .fill {
              fill: ${customColor};
            }
            .stroke {
              stroke: ${customColor};
            }

            .semitransparent {
              fill: ${colors.transparentIconColor || ICON_TRASPARENT_BLUE};
            }
          `}
        >
          <MediumIconStyled>
            <Administration />
          </MediumIconStyled>
        </div>
        {t('common.administration')}
      </H3>
      <AddRoleForm
        locale={locale}
        single={single}
        addRole={addRole}
        moderators={moderators}
        Button={AddModeratorButton}
        addRoleLoading={addRoleLoading}
      />
    </Wrapper>
  );
};

export default React.memo(Header);
