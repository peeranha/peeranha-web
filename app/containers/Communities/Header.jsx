import React from 'react';
import PropTypes from 'prop-types';
import { FormattedMessage } from 'react-intl';
import * as routes from 'routes-config';

import commonMessages from 'common-messages';

import createCommunityIcon from 'images/createCommunity.svg?inline';
import addIcon from 'images/add.svg?external';

import Icon from 'components/Icon';
import TransparentButton from 'components/Button/Contained/Transparent';
import SubHeaderWrapper from 'components/Header/Complex';
import NavigationButton from 'components/Button/Contained/Navigation';
import A from 'components/A';

import messages from './messages';
import languages from './languagesOptions';
import { GO_TO_CREATE_COMMUNITY_SCREEN_BUTTON_ID } from './constants';

const suggestedCommunitiesRoute = routes.suggestedCommunities();
const communitiesRoute = routes.communities();

const Header = ({
  goToCreateCommunityScreen,
  SubHeader,
  changeSorting,
  sorting,
  communitiesNumber,
  setLang,
  language,
}) => {
  const path = window.location.pathname + window.location.hash;

  return (
    <div className="mb-to-sm-0 mb-from-sm-3">
      <SubHeaderWrapper position="top">
        <div>
          <A to={communitiesRoute} href={communitiesRoute}>
            <NavigationButton isLink={path !== communitiesRoute}>
              <FormattedMessage {...commonMessages.communities} />
            </NavigationButton>
          </A>

          <A to={suggestedCommunitiesRoute} href={suggestedCommunitiesRoute}>
            <NavigationButton isLink={path !== suggestedCommunitiesRoute}>
              <FormattedMessage {...messages.voting} />
            </NavigationButton>
          </A>
        </div>

        <div className="right-panel">
          <TransparentButton
            id={`${GO_TO_CREATE_COMMUNITY_SCREEN_BUTTON_ID}_header`}
            onClick={goToCreateCommunityScreen}
            className="d-flex align-items-center"
          >
            <img
              className="d-none d-sm-inline-block"
              src={createCommunityIcon}
              alt="icon"
            />

            <Icon
              className="d-inline-block d-sm-none"
              icon={addIcon}
              width="14"
            />

            <span className="ml-1">
              <FormattedMessage {...messages.suggestCommunity} />
            </span>
          </TransparentButton>
        </div>
      </SubHeaderWrapper>

      <SubHeader
        changeSorting={changeSorting}
        sorting={sorting}
        communitiesNumber={communitiesNumber}
        setLang={setLang}
        language={language}
        languages={languages}
      />
    </div>
  );
};

Header.propTypes = {
  goToCreateCommunityScreen: PropTypes.func,
  SubHeader: PropTypes.any,
  changeSorting: PropTypes.func,
  sorting: PropTypes.object,
  communitiesNumber: PropTypes.number,
  setLang: PropTypes.func,
  language: PropTypes.object,
};

export default Header;
