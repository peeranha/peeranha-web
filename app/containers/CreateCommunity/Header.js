import React from 'react';
import { FormattedMessage } from 'react-intl';

import * as routes from 'routes-config';
import commonMessages from 'common-messages';
import { TEXT_PRIMARY } from 'style-constants';

import createCommunityHeader from 'images/communitiesHeader.svg?inline';
import closeIcon from 'images/closeCircle.svg?inline';

import A from 'components/A';
import H3 from 'components/H3';
import Span from 'components/Span';
import BaseRounded from 'components/Base/BaseRounded';
import { MediumImageStyled } from 'components/Img/MediumImage';

import messages from './messages';

export const Header = /* istanbul ignore next */ () => (
  <BaseRounded className="d-flex align-items-center justify-content-between mb-3">
    <H3 className="d-flex align-items-end">
      <MediumImageStyled src={createCommunityHeader} alt="questionIcon" />
      <FormattedMessage {...messages.newCommunity} />
    </H3>

    <A
      to={routes.communities()}
      href={routes.communities()}
      className="d-inline-flex align-items-center"
    >
      <img className="mr-1" src={closeIcon} alt="x" />
      <Span color={TEXT_PRIMARY}>
        <FormattedMessage {...commonMessages.close} />
      </Span>
    </A>
  </BaseRounded>
);

export default React.memo(Header);
