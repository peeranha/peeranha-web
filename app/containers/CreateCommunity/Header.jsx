import React from 'react';
import { FormattedMessage } from 'react-intl';

import * as routes from 'routes-config';
import commonMessages from 'common-messages';
import { TEXT_PRIMARY } from 'style-constants';

import createCommunityHeader from 'images/communitiesHeader.svg?inline';
import closeIcon from 'images/closeCircle.svg?inline';

import A from 'components/A';
import H3 from 'components/H3';
import Span from 'components/Span';
import Wrapper from 'components/Header/Simple';
import { MediumImageStyled } from 'components/Img/MediumImage';

import messages from './messages';

export const Header = () => (
  <Wrapper className="mb-to-sm-0 mb-from-sm-3">
    <H3>
      <MediumImageStyled src={createCommunityHeader} alt="create-community" />
      <FormattedMessage {...messages.newCommunity} />
    </H3>

    <div className="right-panel">
      <A to={routes.communities()}>
        <button>
          <img className="mr-1" src={closeIcon} alt="x" />
          <Span color={TEXT_PRIMARY}>
            <FormattedMessage {...commonMessages.close} />
          </Span>
        </button>
      </A>
    </div>
  </Wrapper>
);

export default React.memo(Header);
