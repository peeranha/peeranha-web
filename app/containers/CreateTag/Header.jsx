import React from 'react';
import PropTypes from 'prop-types';
import { useTranslation } from 'react-i18next';

import * as routes from 'routes-config';
import { TEXT_PRIMARY, BORDER_PRIMARY } from 'style-constants';

import suggestTagIcon from 'images/tagsHeaderIcon.svg?external';
import closeIcon from 'images/closeCircle.svg?external';

import A from 'components/A';
import H3 from 'components/H3';
import Span from 'components/Span';
import Wrapper, { WrapperRightPanel } from 'components/Header/Simple';
import Icon from 'components/Icon';
import { IconMd } from 'components/Icon/IconWithSizes';
import { MediumIconStyled } from 'components/Icon/MediumIcon';

import { singleCommunityColors } from 'utils/communityManagement';

const colors = singleCommunityColors();

export const Header = ({
  title,
  closeRedirectPage,
  closeButtonAction = null,
}) => {
  const { t } = useTranslation();

  return (
    <Wrapper className="mb-to-sm-0 mb-from-sm-3">
      <H3>
        <MediumIconStyled>
          <Icon
            icon={suggestTagIcon}
            width="43"
            fill={colors.btnColor || BORDER_PRIMARY}
            isColorImportant
          />
        </MediumIconStyled>
        {title || t('tags.newTag')}
      </H3>

      <WrapperRightPanel className="right-panel">
        <A
          to={closeRedirectPage || routes.communities()}
          onClick={closeButtonAction}
        >
          <button>
            <IconMd
              className="mr-1"
              icon={closeIcon}
              fill={colors.btnColor || BORDER_PRIMARY}
              isColorImportant
            />
            <Span
              color={colors.btnColor || TEXT_PRIMARY}
              className="button-label"
            >
              {t('common.close')}
            </Span>
          </button>
        </A>
      </WrapperRightPanel>
    </Wrapper>
  );
};

Header.propTypes = {
  title: PropTypes.element,
  closeRedirectPage: PropTypes.string,
  closeButtonAction: PropTypes.func,
};

export default React.memo(Header);
