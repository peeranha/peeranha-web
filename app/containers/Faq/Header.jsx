import React from 'react';
import * as routes from 'routes-config';
import { FormattedMessage } from 'react-intl';

import commonMessages from 'common-messages';

import { TEXT_PRIMARY } from 'style-constants';
import faqPageHeader from 'images/faqPageHeader.svg?inline';
import infoIcon from 'images/icon-information.svg?inline';

import BaseRounded from 'components/Base/BaseRounded';
import { MediumImageStyled } from 'components/Img/MediumImage';
import H3 from 'components/H3';
import Span from 'components/Span';
import A from 'components/A';

import messages from './messages';

const Header = () => (
  <BaseRounded className="d-flex align-items-center justify-content-between mb-3">
    <H3 className="d-flex align-items-end">
      <MediumImageStyled src={faqPageHeader} alt="faqPageHeader" />
      <FormattedMessage {...messages.faq} />
    </H3>

    <A to={routes.support()} className="d-inline-flex align-items-center">
      <img className="mr-1" src={infoIcon} alt="x" />
      <Span color={TEXT_PRIMARY}>
        <FormattedMessage {...commonMessages.support} />
      </Span>
    </A>
  </BaseRounded>
);

export default React.memo(Header);
