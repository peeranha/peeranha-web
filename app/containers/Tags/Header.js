import React from 'react';
import PropTypes from 'prop-types';
import { FormattedMessage } from 'react-intl';
import * as routes from 'routes-config';
import { TEXT_SECONDARY } from 'style-constants';

import commonMessages from 'common-messages';
import icoTagIcon from 'images/icoTag.svg?inline';
import arrowLeft from 'images/arrowLeft.svg?inline';

import communitiesHeaderFilter from 'images/communitiesHeaderFilter.svg?inline';

import H3 from 'components/H3';
import Dropdown from 'components/Dropdown';
import ArrowIcon from 'components/Dropdown/ArrowIcon';
import Span from 'components/Span';
import Ul from 'components/Ul';
import CheckedItem from 'components/Li/CheckedItem';
import { MediumImageStyled } from 'components/Img/MediumImage';

import Base from 'components/Base';
import NavigationButton from 'components/Button/Contained/Navigation';
import A from 'components/A';

import messages from './messages';
import options from './options';

const tagsRoute = routes.tags();

const H3Styled = H3.extend`
  ${ArrowIcon} {
    transform: scale(0.6);
  }
`;

const Button = /* istanbul ignore next */ ({ sorting }) => (
  <Span className="d-inline-flex align-items-center mr-2 text-capitalize" bold>
    <img
      className="mr-2"
      src={communitiesHeaderFilter}
      alt="communitiesHeaderFilter"
    />
    <FormattedMessage {...options[sorting].message} />
  </Span>
);

const Menu = /* istanbul ignore next */ ({ sortTags, sorting }) => (
  <Ul>
    {Object.keys(options).map(x => (
      <CheckedItem
        key={x}
        data-key={x}
        onClick={sortTags}
        isActive={x === sorting}
      >
        <FormattedMessage {...options[x].message} />
      </CheckedItem>
    ))}
  </Ul>
);

export const Header = /* istanbul ignore next */ ({
  goToCreateTagScreen,
  sortTags,
  sorting,
  currentCommunity,
  tagsNumber,
}) => {
  const path = window.location.pathname + window.location.hash;

  const communityTagsRoute = routes.communityTags(currentCommunity.id);
  const suggestedTagsRoute = routes.suggestedTags(currentCommunity.id);

  return (
    <div>
      <Base
        className="d-flex align-items-center justify-content-between"
        position="top"
      >
        <div>
          <A to={tagsRoute} href={tagsRoute}>
            <NavigationButton className="pl-0" isLink>
              <img className="mr-2" src={arrowLeft} alt="x" />
              <FormattedMessage {...messages.backToList} />
            </NavigationButton>
          </A>

          <A
            to={communityTagsRoute}
            href={communityTagsRoute}
            disabled={path === communityTagsRoute}
          >
            <NavigationButton isLink={path !== communityTagsRoute}>
              <FormattedMessage {...commonMessages.tags} />
            </NavigationButton>
          </A>

          <A
            to={suggestedTagsRoute}
            href={suggestedTagsRoute}
            disabled={path === suggestedTagsRoute}
          >
            <NavigationButton isLink={path !== suggestedTagsRoute}>
              <FormattedMessage {...commonMessages.voting} />
            </NavigationButton>
          </A>
        </div>

        <div>
          <NavigationButton
            onClick={goToCreateTagScreen}
            className="d-inline-flex align-items-center px-0 py-1"
            isLink
          >
            <img className="mr-1" src={icoTagIcon} alt="icoTagIcon" />
            <FormattedMessage {...commonMessages.suggestTag} />
          </NavigationButton>
        </div>
      </Base>

      <Base position="bottom">
        <H3Styled className="d-flex align-items-center justify-content-between">
          <div className="d-flex align-items-center">
            <MediumImageStyled
              src={currentCommunity.avatar}
              alt="communitiesHeader"
            />

            {currentCommunity.name}

            {tagsNumber && (
              <Span
                className="text-lowercase ml-3"
                color={TEXT_SECONDARY}
                fontSize="30"
                bold
              >
                <span>{`${tagsNumber} `}</span>
                <FormattedMessage {...commonMessages.tags} />
              </Span>
            )}
          </div>

          <Dropdown
            button={<Button sorting={sorting} />}
            menu={<Menu sortTags={sortTags} sorting={sorting} />}
            id="tags-dropdown"
            isArrowed
          />
        </H3Styled>
      </Base>
    </div>
  );
};

Button.propTypes = {
  sorting: PropTypes.string,
};

Menu.propTypes = {
  sorting: PropTypes.string,
  sortTags: PropTypes.func,
};

Header.propTypes = {
  currentCommunity: PropTypes.object,
  goToCreateTagScreen: PropTypes.func,
  sortTags: PropTypes.func,
  sorting: PropTypes.string,
  tagsNumber: PropTypes.number,
};

export default React.memo(Header);
