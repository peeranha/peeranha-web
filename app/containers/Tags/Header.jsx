import React from 'react';
import PropTypes from 'prop-types';
import { FormattedMessage } from 'react-intl';
import * as routes from 'routes-config';
import { TEXT_SECONDARY } from 'style-constants';

import commonMessages from 'common-messages';
import icoTagIcon from 'images/icoTag.svg?inline';
import arrowLeft from 'images/arrowLeft.svg?inline';
import addIcon from 'images/addBlue.svg?inline';

import communitiesHeaderFilter from 'images/communitiesHeaderFilter.svg?inline';

import H3 from 'components/H3';
import Dropdown from 'components/Dropdown';
import Span from 'components/Span';
import Ul from 'components/Ul';
import CheckedItem from 'components/Li/CheckedItem';
import Wrapper from 'components/Header/Complex';
import { MediumImageStyled } from 'components/Img/MediumImage';

import NavigationButton from 'components/Button/Contained/Navigation';
import A from 'components/A';

import messages from './messages';
import options from './options';
import { GO_TO_CREATE_TAG_SCREEN_BUTTON_ID } from './constants';

const tagsRoute = routes.tags();

const Button = ({ sorting }) => (
  <Span className="d-inline-flex align-items-center mr-2 text-capitalize" bold>
    <img
      className="mr-2"
      src={communitiesHeaderFilter}
      alt="communitiesHeaderFilter"
    />
    <FormattedMessage {...options[sorting].message} />
  </Span>
);

const Menu = ({ sortTags, sorting }) => (
  <Ul>
    {Object.keys(options).map(x => (
      <CheckedItem
        key={x}
        data-key={x}
        onClick={sortTags}
        isActive={x === sorting}
      >
        <FormattedMessage {...options[x].message} />
      </CheckedItem>
    ))}
  </Ul>
);

export const Header = ({
  goToCreateTagScreen,
  sortTags,
  sorting,
  currentCommunity,
  tagsNumber,
}) => {
  const path = window.location.pathname + window.location.hash;

  const communityTagsRoute = routes.communityTags(currentCommunity.id);
  const suggestedTagsRoute = routes.suggestedTags(currentCommunity.id);

  return (
    <div className="mb-to-sm-0 mb-from-sm-3">
      <Wrapper position="top">
        <div>
          <A to={tagsRoute}>
            <NavigationButton className="pl-0" isLink>
              <img src={arrowLeft} alt="x" />
              <span className="d-none d-sm-inline ml-2">
                <FormattedMessage {...messages.backToList} />
              </span>
            </NavigationButton>
          </A>

          <A to={communityTagsRoute} disabled={path === communityTagsRoute}>
            <NavigationButton isLink={path !== communityTagsRoute}>
              <FormattedMessage {...commonMessages.tags} />
            </NavigationButton>
          </A>

          <A to={suggestedTagsRoute} disabled={path === suggestedTagsRoute}>
            <NavigationButton isLink={path !== suggestedTagsRoute}>
              <FormattedMessage {...commonMessages.voting} />
            </NavigationButton>
          </A>
        </div>

        <div className="right-panel">
          <NavigationButton
            data-communityid={currentCommunity.id}
            onClick={goToCreateTagScreen}
            id={`${GO_TO_CREATE_TAG_SCREEN_BUTTON_ID}_header`}
            className="d-inline-flex align-items-center px-0 py-1"
            isLink
          >
            <img
              className="d-none d-sm-inline-block"
              src={icoTagIcon}
              alt="icon"
            />
            <img
              className="d-inline-block d-sm-none"
              src={addIcon}
              alt="icon"
            />
            <span className="ml-1">
              <FormattedMessage {...commonMessages.suggestTag} />
            </span>
          </NavigationButton>
        </div>
      </Wrapper>

      <Wrapper position="bottom">
        <H3>
          <MediumImageStyled src={currentCommunity.avatar} alt="tags" />

          {currentCommunity.name}

          {tagsNumber && (
            <Span
              className="d-none d-sm-inline text-lowercase ml-3"
              color={TEXT_SECONDARY}
              fontSize="30"
              bold
            >
              <span>{`${tagsNumber} `}</span>
              <FormattedMessage {...commonMessages.tags} />
            </Span>
          )}
        </H3>

        <div className="right-panel">
          <Dropdown
            button={<Button sorting={sorting} />}
            menu={<Menu sortTags={sortTags} sorting={sorting} />}
            id="tags-dropdown"
            isArrowed
          />
        </div>
      </Wrapper>
    </div>
  );
};

Button.propTypes = {
  sorting: PropTypes.string,
};

Menu.propTypes = {
  sorting: PropTypes.string,
  sortTags: PropTypes.func,
};

Header.propTypes = {
  currentCommunity: PropTypes.object,
  goToCreateTagScreen: PropTypes.func,
  sortTags: PropTypes.func,
  sorting: PropTypes.string,
  tagsNumber: PropTypes.number,
};

export default React.memo(Header);
