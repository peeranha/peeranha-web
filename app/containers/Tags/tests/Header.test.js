import { Header } from '../Header';

describe('Header', () => {
  const props = {
    goToCreateTagScreen: jest.fn(),
    sortTags: jest.fn(),
    sorting: 'id',
    currentCommunity: {},
    tagsNumber: 1,
  };

  it('test', () => {
    expect(Header(props)).toMatchSnapshot();
  });
});
