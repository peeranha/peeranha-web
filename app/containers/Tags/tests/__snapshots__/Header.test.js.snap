// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header test 1`] = `
<div
  className="mb-to-sm-0 mb-from-sm-3"
>
  <Base
    position="top"
  >
    <div>
      <A__ALink
        to="/tags"
      >
        <Button
          className="pl-0"
          isLink={true}
        >
          <img
            alt="x"
            src="IMAGE_MOCK"
          />
          <span
            className="d-none d-sm-inline ml-2"
          >
            <FormattedMessage
              id="app.containers.Tags.backToList"
              values={Object {}}
            />
          </span>
        </Button>
      </A__ALink>
      <A__ALink
        disabled={false}
        to="/communities/undefined/tags"
      >
        <Button
          isLink={true}
        >
          <FormattedMessage
            id="app.common.tags"
            values={Object {}}
          />
        </Button>
      </A__ALink>
      <A__ALink
        disabled={false}
        to="/communities/undefined/tags/suggested"
      >
        <Button
          isLink={true}
        >
          <FormattedMessage
            id="app.common.voting"
            values={Object {}}
          />
        </Button>
      </A__ALink>
    </div>
    <div
      className="right-panel"
    >
      <Button
        className="d-inline-flex align-items-center px-0 py-1"
        id="app_containers_GO_TO_CREATE_TAG_SCREEN_BUTTON_ID_header"
        isLink={true}
        onClick={[MockFunction]}
      >
        <img
          alt="icon"
          className="d-none d-sm-inline-block"
          src="IMAGE_MOCK"
        />
        <img
          alt="icon"
          className="d-inline-block d-sm-none"
          src="IMAGE_MOCK"
        />
        <span
          className="ml-1"
        >
          <FormattedMessage
            id="app.common.suggestTag"
            values={Object {}}
          />
        </span>
      </Button>
    </div>
  </Base>
  <Base
    position="bottom"
  >
    <Span>
      <Img
        alt="tags"
      />
      <Span
        bold={true}
        className="d-none d-sm-inline text-lowercase ml-3"
        color="#7b7b7b"
        fontSize="30"
      >
        <span>
          1 
        </span>
        <FormattedMessage
          id="app.common.tags"
          values={Object {}}
        />
      </Span>
    </Span>
    <div
      className="right-panel"
    >
      <UNDEFINED
        button={
          <Button
            sorting="id"
          />
        }
        id="tags-dropdown"
        isArrowed={true}
        menu={
          <Menu
            sortTags={[MockFunction]}
            sorting="id"
          />
        }
      />
    </div>
  </Base>
</div>
`;
